upup/pkg/fi/cloudup/loader.go

```diff
@@ -202,7 +203,10 @@ func (l *Loader) addAssetCopyTasks(assets []*assets.Asset) error {
 				Name:        fi.String(asset.DockerImage),
 				SourceImage: fi.String(asset.CanonicalLocation),
 				TargetImage: fi.String(asset.DockerImage),
+				// TODO
```

Comment:
Suggest threading it through BuildTasks, but you can leave a TODO for me if it gets messy